ReactiveTask/Errors.swift

```diff
@@ -12,7 +12,7 @@ import ReactiveCocoa
 /// An error originating from ReactiveTask.
 public enum TaskError: ErrorType {
 	/// A shell task exited unsuccessfully.
-	case ShellTaskFailed(exitCode: Int32, standardError: String?)
+	case ShellTaskFailed(task: Task, exitCode: Int32, standardError: String?)
```

Comment:
Can we remove the `task:` label from the `Task` argument? The meaning of the argument should be obvious, so I don't think the label is necessary.